panther_analysis_tool/backend/public_api_client.py

```diff
@@ -154,7 +153,9 @@ def feature_flags_query(self) -> DocumentNode:
 
     def _load(self, name: str) -> DocumentNode:
         if name not in self._cache:
-            self._cache[name] = Path(_get_graphql_content_filepath(name)).read_text()
+            file_path = _get_graphql_content_filepath(name)
```

Comment:
Can you elaborate on this change?

Reply:
I thought I needed `with open` to specify the file encoding; turns out that's not the case.

Reverted and updated 👍🏻